src/containers/Nav/NavNotificationBar.tsx

```diff
@@ -13,10 +13,18 @@ const throwMockError = () => {
   }
 }
 
-export const FeedBackButton = () => {
+type FeedBackButtonProps = {
```

Comment:
On utilise des interfaces par default (ici ça ne change rien mais c'est pour avoir un code)

```suggestion
interface FeedBackButtonProps {
```